pkg/pci/pci_linux.go

```diff
@@ -44,6 +44,28 @@ func getDeviceModaliasPath(ctx *context.Context, address string) string {
 	)
 }
 
+func getDeviceRevision(ctx *context.Context, address string) string {
+	paths := linuxpath.New(ctx)
+	pciAddr := AddressFromString(address)
+	if pciAddr == nil {
+		return ""
+	}
+	revisionPath := filepath.Join(
+		paths.SysBusPciDevices,
+		pciAddr.Domain+":"+pciAddr.Bus+":"+pciAddr.Slot+"."+pciAddr.Function,
```

Comment:
this suggests we may want to add a `String` method to `pci.Address` (not in this PR)